README.rst

```diff
@@ -1,3 +1,8 @@
+:**NOTE**: **This github repository is archived. The repository contents and history have moved to** `google-cloud-python`_.
+
+.. _google-cloud-python: https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-build
```

Comment:
We're going from `python-cloudbuild` (no separator in `cloudbuild`) to `google-cloud-build` (with a separator in `cloud-build`. Not a problem, right?)

Reply:
`python-cloudbuild` is the repository name, which is unrelated to the client library code.  `google-cloud-build` is the name of the package that is published to PyPI and we will continue to use the same package name in the monorepo.

https://github.com/googleapis/python-cloudbuild/blob/1d2376a02917cd1fcf21aa9415cf15d7b46e9ede/setup.py#L23

https://github.com/googleapis/google-cloud-python/blob/8c78ad90c4f2bfe9414a4f60f0022ccdcdeb758f/packages/google-cloud-build/setup.py#L23